core/src/main/resources/schema/infinispan-config-15.0.xsd

```diff
@@ -1072,6 +1072,9 @@
                 The value can be a relative or absolute path. Relative paths are created
                 relative to the configured global persistent location, or to the current
                 working directory when global state is disabled.
+                By default, the cache name is used as a relative path for index path.
+                When the value is set, we need to use different paths for different caches,
+                it allows different caches to work safely with the same indexed entities (index names).
```

Comment:
This text (and its copy in the asciidoc below) is confusing. 
Maybe something like:

When setting a custom value, ensure that there are no conflicts between caches using the same indexed entities.